experimental/dotnet_core_runtime/README.md

```diff
@@ -0,0 +1,88 @@
+# Microsoft .NET Core Distroless Container Image
+
+This project defines a minimal Linux with [.NET Core](https://github.com/dotnet/core).
+
+Specifically, the image contains everything described in the [../../cc/README.md](../../cc/README.md), and includes following prerequisites.
+
+## NET Core 3.1 Runtime and its Dependencies
+
+- [3.1/runtime-deps/buster-slim/amd64/Dockerfile](https://github.com/dotnet/dotnet-docker/blob/master/3.1/runtime-deps/buster-slim/amd64/Dockerfile)
+- [3.1/runtime/buster-slim/amd64/Dockerfile](https://github.com/dotnet/dotnet-docker/blob/master/3.1/runtime/buster-slim/amd64/Dockerfile)
+
+```text
```

Comment:
Since this is a long list, this can easily go out of sync or change going forward. Library versions will also vary depending on whether `-debian9`, `-debian10`, `-debian11`, etc. I think we can instead put a link to the Bazel file instead of explicitly listing debs here.

Reply:
I followed the content from existing [README.md](https://github.com/GoogleContainerTools/distroless/blob/master/experimental/dotnet/README.md#image-contents).

Reply:
Thank you. PR updated.